eessi_container.sh

```diff
@@ -408,6 +408,19 @@ else
   echo "Using ${EESSI_HOST_STORAGE} as tmp directory (to resume session add '--resume ${EESSI_HOST_STORAGE}')."
 fi
 
+# if ${RESUME} is a file, unpack it into ${EESSI_HOST_STORAGE}
+if [[ ! -z ${RESUME} && -f ${RESUME} ]]; then
+  if [[ "${RESUME}" == *.tgz ]]; then
+    tar xf ${RESUME} -C ${EESSI_HOST_STORAGE}
+  # Add support for resuming from zstd-compressed tarballs
+  elif [[ "${RESUME}" == *.zst && -x "$(command -v zstd)" ]]; then
```

Comment:
Why not check once if `zstd` is supported, define a constant, and use that everywhere:
```shell
if [[ -x $(command -v zstd) ]]; then
    USE_ZSTD=true
else
    USE_ZSTD=false
fi

Reply:
well, you can, but then you have to check if [[ `USE_ZSTD` == 'true' ]] (remember, this is bash, not python, there is no boolean in bash.

Since you have to do a test anyway, the current is explicit, equally short, and execution time of `command -v` is negligible. No point in storing it in an environment variable, then having to remember what the value to check for is (true or True or whatever).